Modules/ITS/src/ITSFeeTask.cxx

```diff
@@ -408,6 +520,11 @@ void ITSFeeTask::monitorData(o2::framework::ProcessingContext& ctx)
     }
   }
 
+  for (int ierror = 0; ierror < o2::itsmft::GBTLinkDecodingStat::NErrorsDefined; ierror++) {
+    int feeError = mLinkErrorVsFeeid->Integral(1, mLinkErrorVsFeeid->GetXaxis()->GetNbins(), ierror + 1, ierror + 1);
+    mErrorPlots->SetBinContent(ierror + 1, feeError);
```

Comment:
Is it expected that we have TH1 with the number of decoding errors only for  GBTLinkDecodingStat? @iravasen